docs/extensions/FrameworkElementEx.md

```diff
@@ -0,0 +1,42 @@
+---
+title: FrameworkElement extentions
+author: ST-Apps
+ms.date: 01/05/2018
+description: FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, FrameworkElement, extentions
+---
+
+# FrameworkElement extentions
```

Comment:
As a title `Extensions` should be capitalized.

Reply:
@michael-hawker fixed.